rwth_upper_body_skeleton_random_walk/launch/fixed.launch

```diff
@@ -0,0 +1,20 @@
+<launch>
+
+<include file ="$(find ground_plane_estimation)/launch/ground_plane_fixed.launch"/>
+<include file ="$(find upper_body_detector)/launch/upper_body_detector.launch"/>
```

Comment:
Please do not launch the ground plane and upper body detector in here to make it possible to have it running together with the overall people perception pipeline.